tests/phpunit/tests/post/wpTrashPost.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
```

Comment:
I think we need a docblock here:
```
/**
 * @group post
 * @covers ::wp_trash_post
 */
```
See https://github.com/WordPress/wordpress-develop/blob/trunk/tests/phpunit/tests/post/wpPublishPost.php